ruby/conclusion/project_ruby_final.md

```diff
@@ -16,12 +16,14 @@ This is a great project to have as a part of your portfolio going forward becaus
   3. Make it so you can save the board at any time (remember how to serialize?)
   2. Write tests for the important parts.  You don't need to TDD it (unless you want to), but be sure to use RSpec tests for anything that you find yourself typing into the command line repeatedly.
   3. Do your best to keep your classes modular and clean and your methods doing only one thing each.  This is the largest program that you've written, so you'll definitely start to see the benefits of good organization (and testing) when you start running into bugs.
-  4. Have fun!  Check out the [unicode characters](http://en.wikipedia.org/wiki/Chess_symbols_in_Unicode) for a little spice for your gameboard.
-  5. (Optional extension) Build a very simple AI computer player (perhaps who does a random legal move)
+  4. Unfamiliar with Chess? Check out some of the additional resources to help you get your bearings.
+  5. Have fun!  Check out the [unicode characters](http://en.wikipedia.org/wiki/Chess_symbols_in_Unicode) for a little spice for your gameboard.
+  6. (Optional extension) Build a very simple AI computer player (perhaps who does a random legal move)
 </div>
 
 ### Additional Resources
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
 * [Illustrated rules of Chess](http://www.chessvariants.org/d.chess/chess.html)
+* [Chess notation](https://en.wikipedia.org/wiki/Chess_notation)
```

Comment:
I'd consider expanding the description here to give a bit more background / context, but not a blocker for me

Reply:
@ChargrilledChook Any specific suggestions on how you would want to see this expanded? 

I'm also good with it as is since I think the link text is very clear. 

But I could also see something like: "Chess notation for humans and computers"

That might give some idea for learners to think about the differences between notation for humans vs. computers and how they might utilize one or the other.

Reply:
I like that :ok_hand: but it's a fairly minor detail either way

Reply:
> @ChargrilledChook Any specific suggestions on how you would want to see this expanded?
> 
> I'm also good with it as is since I think the link text is very clear.
> 
> But I could also see something like: "Chess notation for humans and computers"
> 
> That might give some idea for learners to think about the differences between notation for humans vs. computers and how they might utilize one or the other.

I like that a lot, and I will push a commit to update the text of the link to "Chess notation for humans and computers"